lib/migration.js

```diff
@@ -9,35 +9,43 @@ var async = require('async');
 module.exports = mixinMigration;
 
 function mixinMigration(Oracle) {
+  Oracle.prototype.showFields = function(model, cb) {
+    var sql = 'SELECT column_name AS "column", data_type AS "type",' +
+      ' nullable AS "nullable"' + // , data_default AS "Default"'
+      ' FROM "SYS"."USER_TAB_COLUMNS" WHERE table_name=\'' +
+      this.table(model) + '\'';
+    this.execute(sql, function(err, fields) {
+      if (err)
+        return cb(err);
+      else
+        cb(err, fields);
+    });
+  };
+
   /*!
    * Discover the properties from a table
    * @param {String} model The model name
    * @param {Function} cb The callback function
    */
   Oracle.prototype.getTableStatus = function(model, cb) {
-    function decoratedCallback(err, data) {
-      if (err) {
-        console.error(err);
-      }
-      if (!err) {
-        data.forEach(function(field) {
-          field.type = mapOracleDatatypes(field.type);
```

Comment:
Fixed in commit https://github.com/strongloop/loopback-connector-oracle/pull/110/commits/c006e7dbd1780428600bdaae3e2eb82890230110